tests/NodeTypeDiscovery/NodeDefinitionTest.php

```diff
@@ -11,24 +11,52 @@
 
 namespace PHPCR\Tests\NodeTypeDiscovery;
 
+use PHPCR\NodeType\NodeDefinitionInterface;
+use PHPCR\NodeType\NodeTypeInterface;
+use PHPCR\NodeType\NodeTypeManagerInterface;
+
 /**
- * Test the NoteDefinition §8.
+ * Test NodeDefinition behaviour and reading NodeDefinition from NodeTypeDefinition §8.
  *
  * Requires that NodeTypeManager->getNodeType and NodeTypeDefinition->getChildNodeDefinitions() works correctly
  */
 class NodeDefinitionTest extends \PHPCR\Test\BaseCase
 {
     private static $base;
+
+    /**
+     * @var NodeTypeInterface
+     */
     private static $file;
+
+    /**
+     * @var NodeTypeInterface
+     */
     private static $folder;
+
+    /**
+     * @var NodeTypeInterface
+     */
     private static $hierarchyNodeType;
-    /** jcr:content of nt:file */
+
+    /**
+     * Node definition of the jcr:content in an nt:file type.
+     *
+     * @var NodeDefinitionInterface
+     */
     private $content;
+
+    /**
+     * Node definition of a hierarchy node.
+     *
+     * @var NodeDefinitionInterface
+     */
     private $hierarchyNodeDef;
 
     public static function setupBeforeClass($fixtures = false)
```

Comment:
the problem in jackalope-doctrine-dbal was that we have no fixtures we load for this test. in jackrabbit, fixtures are not removed in this case so we did not see the mistake. separated the tests now.